README.md

```diff
@@ -1,9 +1,13 @@
-# Airflow with Docker
+# Airflow with ~~Docker~~ Kubernetes
 This image builds upon [Puckel's Airflow image](https://github.com/puckel/docker-airflow), specialising it for use as 
-controller of tasks being run in standalone Docker containers (using the Python Docker bindings).
+controller of tasks being run ~~standalone Docker containers (using the Python Docker bindings)~~ in a Kubernetes cluster.
 
-It uses a LocalExecutor to allow for task parallelism, doesn't laod the example, and adds a few utilities as well as
-localising to South Africa, otherwise the config is left as the defaults.
+Please note these instructions are for launching the Docker container standalone. Please head over to the [Kubernetes README](./k8s/README.md)
```

Comment:
"The instructions in this README are for launching this docker image in standalone (i.e. non-Kubernetes) mode."

Reply:
Just trying to be super explicit because every little bit of specificity helps in this domain.

Reply:
sure, will make it more specific.